packages/loader-react/src/Loader.tsx

```diff
@@ -1,16 +1,24 @@
 import React from "react";
 interface Props {
+    textDescription?: string;
     negative?: boolean;
     inline?: boolean;
     className?: string;
 }
-export const Loader = ({ negative = false, inline = false, className }: Props) => {
+export const Loader = ({ textDescription = "Laster inn", negative = false, inline = false, className }: Props) => {
```

Comment:
An option to consider is making the prop mandatory, with no default, to force devs/designers to think about what the loader represents.

Reply:
Yeah, I like that. Also, we probably should have a pattern on how to use loaders. I think we should try to avoid loaders, unless the wait time is more than say 500 ms. And then the loader should show for at least 500 ms.